opacclient/opacapp/src/main/res/layout/content_accountitem_detail.xml

```diff
@@ -90,7 +90,7 @@
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:padding="16dp"
-                    android:visibility="@{item.author != null ? View.VISIBLE : View.GONE}">
+                    android:visibility="@{(item.author != null &amp;&amp; !item.author.equals(&quot;&quot;)) ? View.VISIBLE : View.GONE}">
```

Comment:
You're right (although that should be a rare case limited to the weird string parsing in Biber which I used to test). Should've fixed it in Biber, will fix it in both places.